H4LCppModule.py

```diff
@@ -261,6 +272,7 @@ def beginFile(self, inputFile, outputFile, inputTree, wrappedOutputTree):
         # Branches for 2l2nu channel: ZZ kinematics
         self.out.branch("HZZ2l2nu_ZZmT",  "F")
         self.out.branch("HZZ2l2nu_ZZpT",  "F")
+        self.out.branch("Pz_neutrino", "F")
```

Comment:
`Pz_neutrino` is coming from GEN info. Right? 

If so, then change name to `GEN_Pz_Neutrino`.